functions/bigtable/requirements.txt

```diff
@@ -1 +1,2 @@
-google-cloud-bigtable==2.19.0
+functions-framework==3.*
```

Comment:
Can we pin to a specific version? We have bots to automatically upgrade versions. `functions-framework` doesn't usually cause any issues when upgrading.

Reply:
This seems to conflict with the official cloud functions docs:
https://cloud.google.com/functions/docs/create-deploy-http-python#specify_dependencies

Reply:
After chatting with @daniel-sanche, I hardcoded the actual version